scripts/webidl/convert.js

```diff
@@ -46,4 +46,4 @@ function onlyIDL(filePath) {
   return path.extname(filePath) === ".idl";
 }
 
-doConversion("lib/jsdom/living/generated/events").done();
+doConversion("lib/jsdom/living/events").done();
```

Comment:
It does, but in concats into a single "source file" which is then used for generation and still needs to be able to find the right implementation directory, which is then not possible anymore since its regarded as a single entity.